self-service-kiosk/src/app/config/catalog-config.js

```diff
@@ -0,0 +1,19 @@
+export default {
+
+    'fm': {
+        fetchMenus: async () => {
+            // talk to bff
+            let bffPort = 3000;
```

Comment:
Shouldn't the port depends on `process.env.PORT` and only have "3000" as fallback?